Makefile

```diff
@@ -17,14 +17,19 @@
 .PHONY: help purge install uninstall clean test-dependencies lint-server lint-ui lint yarn-install eslint-ui eslint-check-ui prettier-ui prettier-check-ui flake lint-server-dependencies dev-link dev-unlink
 .PHONY: build-ui build-server install-server watch install-extensions build-jupyterlab install-server-package check-install only-install-server
 .PHONY: test-server test-ui test-integration test-integration-debug test docs-dependencies docs dist-ui release pytest
-.PHONY: validate-runtime-images elyra-image publish-elyra-image kf-notebook-image
+.PHONY: validate-runtime-images elyra-image publish-elyra-image kf-notebook-image elyra-dev-images
```

Comment:
Do we need to introduce another task? Can't we handle this based on the tag in use?

Reply:
Modified the existing dockerfiles to accept `local` as a tag and to builds using the local wheel dist.